### ISSUE TITLE
Full screen button background is grey

### ISSUE THREAD

**Issue by barishshalom** [JakeSteam/Vidsta#4]
The fullscreen button background is grey please set its background to null or transperant
Thank you

**Comment by JakeSteam** [comment]
Hey,

As mentioned in the docs, you can override the button with your own.

Jake

**Comment by barishshalom** [comment]
Yes I know you can change the resource, but you can't change the background.
Can you point me to how to change the background?
Thank you

**Comment by JakeSteam** [comment]
Ah, I see. It's not a feature currently, sorry.

**Comment by amank22** [comment]
Yes the background is indeed grey and even on changing resource the background remains grey.
